.travis.yml

```diff
@@ -28,7 +31,11 @@ services:
   - redis-server
 
 before_install:
+  # If necessary, set up an appropriate version of pypy.
+  - if [ ! -z "$PYPY_VERSION" ]; then source utils/setup-pypy-travis.sh; fi
+  - if [ ! -z "$PYPY_VERSION" ]; then python --version 2>&1 | fgrep "PyPy $PYPY_VERSION"; fi
```

Comment:
To make sure we're using the correct version. If the `fgrep` fails, Travis sees it as an error and aborts the build.

If we fail to set up pyenv correctly, it's possible for us to still be in the original virtualenv with the wrong Python.